spec/system/work_upload_max_spec.rb

```diff
@@ -1,43 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Removed this test because we now let Uppy handle the validation for the maximum number of files to upload instead of rolling our own.